sdk/go/common/testing/environment.go

```diff
@@ -129,7 +129,9 @@ func (e *Environment) ImportDirectory(path string) {
 func (e *Environment) DeleteEnvironment() {
 	e.Helper()
 	err := os.RemoveAll(e.RootPath)
-	assert.NoErrorf(e, err, "cleaning up test directory %q", e.RootPath)
+	if err != nil {
```

Comment:
Actually, would it be worth a comment, so someone doesn't change this back to an assert?

Reply:
ack